static/js/components/CourseList.js

```diff
@@ -119,7 +120,7 @@ class CourseList extends React.Component {
             <br/>
             <p>You need to pass {totalCourses - coursesPassed} more courses
               before you can apply for the <strong>{program.title}</strong> Master’s Degree.</p>
-            <MuiThemeProvider>
```

Comment:
I don't believe we need this line at all... this is done in `dashboard_routes.js`. Can you see if it still works if we remove it?